controls/cis_ocp_1_4_0/section-5.yml

```diff
@@ -138,28 +138,32 @@ controls:
       level: level_2
   - id: '5.7'
     title: General Policies
-    status: pending
+    status: partial
     rules: []
     controls:
     - id: 5.7.1
       title: Create administrative boundaries between resources using namespaces
-      status: pending
-      rules: []
+      status: partial
```

Comment:
Why are all the statuses partial?

Reply:
Reviewed these again and updated them to manual since the rules don't check the cluster itself.